deployment/app.fly.dockerfile

```diff
@@ -21,16 +21,23 @@ RUN apt-get update && apt-get install nodejs -y
 # Install Yarn Package Manager
 RUN npm install --global yarn
 
-COPY todo0 todo0
-COPY wiki0 wiki0
+# Copy packages
+COPY todo0 packages/todo0
```

Comment:
curious, why do we move these folders into `/packages` for fly.io?  Should we update the code directly to live under `/packages`?

Reply:
The committed package.json workspaces block has folders that won't exist on fly.io since we copy specific folders. Instead of rewriting the block with jq to have those specific folders, I thought it would be easier to lump them into packages and the rewrite is static and easy to read.

Definitely aim to make the repo match this so we can get rid of that intermediate hack on the deployment docker file.

Reply:
👍 makes sense